src/util/searchSubmissions.js

```diff
@@ -29,18 +59,20 @@ async function searchSubmissions(
                 .then((entries) => {
                     entries.forEach((doc) => {
                         res.push(new StudentSubmission(
-                            doc.data().id, doc.data().subjectName, doc.data().location, 
-                            doc.data().date, doc.data().description, doc.data().sources,
+                            doc.data().id, 
+                            doc.data().subjectName,
+                            doc.data().location, 
+                            doc.data().date,
+                            doc.data().description,
+                            doc.data().sources,
                             `${doc.data().studentFirst} ${doc.data().studentLast}`,
                             doc.data().submittedDate,
                             // TODO: store student's class
```

Comment:
so are we still doing this todo? maybe talk to @kzhang01 @namratarajaraman. I remember that we removed this student class though so we should remove this TODO

If we still have it as a TODO, can we do it? should be quick